### PR TITLE
Chat History changes

### PR DESCRIPTION
**Pull request by JasonSowers**
Based on Bill's feedback. @danmarshall please review and let me know if there is more I can do to help.

### CODE REVIEW

File: src/Chat.tsx [hunk 1]
```diff
@@ -76,8 +76,8 @@ export class Chat extends React.Component<ChatProps, {}> {
         konsole.log("BotChat.Chat props", props);
 
         if (props.pastHistory) {
-            this.store.dispatch<HistoryAction>({
-                type: 'Past_History',
+            this.store.dispatch<ChatAction>({
```
Comment (danmarshall) [inline src/Chat.tsx]: this is still a HistoryAction

File: src/Chat.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ export interface ChatProps {
     directLine?: DirectLineOptions,
     speechOptions?: SpeechOptions,
     locale?: string,
-    pastHistory?: Activity[],
+    history?: Activity[],
```
Comment (danmarshall) [inline src/Chat.tsx:29]: there's a lot more places to rename. See the original PR

File: README.md [hunk 3]
```diff
@@ -222,10 +222,11 @@ You can supply WebChat with the id (and, optionally, a friendly name) of the cur
...
+Using a custom client you can store conversation history to then retrieve and send the conversation history to WebChat. You can initialize the WebChat conversation by passing an array of activity objects in `pastHistory: [ activity_object1, activity_object2, ... ]` to `App`/`Chat`. Each activity object should have a unique `id` value, and the `from` property should match either a user identity (above) or the bot's identity. An example activity message might look like:
 ```
 { id: <unique id>, from: { id: <userid>, name: <username> }, type: "message", text: "The user said this in the past." }
 ```
+The fields above are the fields required to provide chat history to webchat.  Keep in mind you will probably want to store more information.  Specifically you will most likely want a timestamp on each activity to order them in the correct chronological order.  It is suggested to consult the [Activity object](https://docs.microsoft.com/en-us/bot-framework/rest-api/bot-framework-rest-connector-api-reference#activity-object) documentation when storing activities for the purnpose of displaying chat history.
```
Comment (danmarshall) [inline README.md]: typo - "purnpose"

Comment (danmarshall) [reply README.md]: Also, I am not certain that we are ordering by timestamp, I will need to look.

File: src/Chat.tsx [hunk 4]
```diff
@@ -76,9 +76,9 @@ export class Chat extends React.Component<ChatProps, {}> {
         konsole.log("BotChat.Chat props", props);
 
         if (props.pastHistory) {
```
Comment (danmarshall) [inline src/Chat.tsx]: there's still a "pastHistory" on this line. You may want to do a global search, or use VSCode's "rename" feature.

File: src/Chat.tsx (hunk 2 above)
Comment (JasonSowers) [reply src/Chat.tsx:29]: I only see the 3 in Chat.tsx am I completely missing something?

Comment (danmarshall) [reply src/Chat.tsx:29]: see the files in the /test/ folder in the PR

**Pull request merged**